spec.md

```diff
@@ -22,11 +21,9 @@
   - [Broker Errors](#broker-errors)
   - [Orphans](#orphans)
 
-<div id="changes"/> 
-## Changes ##
+## <div id="changes"/> Changes
```

Comment:
@duglin and @angarg12 thanks both. changes made per your feedback - this last commit includes the removal of the divs and minor changes to make the markdown anchors work properly. if you spot anything else, pls don't hesitate to get in touch. thanks again.